docs/other-sdks.md

```diff
@@ -0,0 +1,17 @@
+---
+id: other-sdks
+title: Other SDKs
+sidebar_label: Other SDKs
+---
+
+There is demand for other Temporal SDKs other than Go and Java. The user community has taken initiative to build some of these, but please note that they are not officially supported or endorsed by Temporal unless explicitly mentioned:
+
+- [Ruby SDK by Coinbase](https://github.com/coinbase/temporal-ruby)
```

Comment:
Should we be linking to these? Is there any other context we can provide here about the status ?

Reply:
i dont know but i think it looks good to link. since everything on this page will not be "officially supported" but we can still acknowledge the work being done by the community. @rylandg ?

Reply:
Cool - sgtm then. Just curious if we plan to fork it and maintain?

Reply:
for Ruby, no idea, but i'd guess its secondary compared to python/nodejs (since we probably trust Coinbase's impl)